### ISSUE TITLE
请问

### ISSUE THREAD

**Issue by DuGuyuanqing** [SaulZhang/Short-Text-classification#1]
我该如何得到x_train_dataset.pkl，y_train_dataset.pkl， x_test_dataset.pkl，y_test_dataset.pkl呢？感谢您的解答

**Comment by SaulZhang** [comment]
Hi, You can attain the public dataset from the website in the readme.md, and then split the dataset into training set and test set.
